### PR TITLE
Add correct photo attribution for horse selfies

### PR DESCRIPTION
**Pull request by joms**
Also, use appropriate emoji for style

### CODE REVIEW

**Comment by joms** [comment]
Note: This has not been tested, nor should it

**Comment by joms** [comment]
Because of missing testing, this will fail

**Comment by joms** [comment]
Updated with less testing and less phails

File: index.html [hunk 1]
```diff
@@ -186,7 +186,7 @@ <h4 id="attribution"></h4>
 	var at = document.getElementById("attribution");
 	var n = parseInt(1+Math.random()*13);
 	horse.src = horsyHorse.horse;
-	at.innerHTML = "Picture attribution: " + horsyHorse.attribution;
+	at.innerHTML = horsyHorse.attribution === "N/A" ? "📸: " + horsyHorse.attribution : "📸: Horse";
```
Comment (KristianLyng) [inline index.html:189]: Does that utf-sign render properly? Doesn't render at all in github/chromium....

**Pull request merged**